Sources/Actions/Admin/SearchEngines.php

```diff
@@ -439,7 +439,8 @@ public function logs(): void
 
 					Utils::$context['spider_logs']['rows'][$k]['data']['viewing']['class'] = $new_url['class'];
 				} else {
-					Utils::$context['spider_logs']['rows'][$k]['data']['viewing']['value'] = $new_url;
+					// @TODO: Indirect modification of overloaded element of SMF\ItemList has no effect in
+					@Utils::$context['spider_logs']['rows'][$k]['data']['viewing']['value'] = $new_url;
```

Comment:
I didn't want to touch this right now, but I got that error.  We will need to rework the logic.